src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisService.java

```diff
@@ -57,8 +51,15 @@ public Mono<UUID> runAndSaveResult(SecurityAnalysisRunContext runContext) {
 
         // update status to running status
         return setStatus(List.of(resultUuid), SecurityAnalysisStatus.RUNNING.name()).then(
-                Mono.fromRunnable(() ->
-                        sendRunMessage(new SecurityAnalysisResultContext(resultUuid, runContext).toMessage(objectMapper)))
+                Mono.fromRunnable(() -> {
```

Comment:
Why do you remove the `SecurityAnalysisResultContext::toMessage` ?

Reply:
I create the message directly in NotificationService.

Reply:
I moved back the creation of the message to SecurityAnalysisResultContext